Sources/Rendering/Core/AxesActor/index.js

```diff
@@ -107,76 +123,88 @@ function vtkAxesActor(publicAPI, model) {
   publicAPI.update();
   const _debouncedUpdate = macro.debounce(publicAPI.update, 0);
 
-  const { setConfig, setXAxisColor, setYAxisColor, setZAxisColor } = publicAPI;
-
-  publicAPI.setConfig = (c) => {
-    if (setConfig(c)) {
-      _debouncedUpdate();
-      return true;
-    }
-    return false;
+  publicAPI.setXAxisColor = (color) => {
+    model.xConfig.color = color;
+    _debouncedUpdate();
   };
 
-  publicAPI.setXAxisColor = (c) => {
-    if (setXAxisColor(c)) {
-      _debouncedUpdate();
-      return true;
-    }
-    return false;
+  publicAPI.setYAxisColor = (color) => {
+    model.yConfig.color = color;
+    _debouncedUpdate();
   };
 
-  publicAPI.setYAxisColor = (c) => {
-    if (setYAxisColor(c)) {
-      _debouncedUpdate();
-      return true;
-    }
-    return false;
+  publicAPI.setZAxisColor = (color) => {
+    model.zConfig.color = color;
+    _debouncedUpdate();
   };
 
-  publicAPI.setZAxisColor = (c) => {
-    if (setZAxisColor(c)) {
+  publicAPI.getXAxisColor = (color) =>
+    model.xConfig?.color || model.config?.color;
+
+  publicAPI.getYAxisColor = (color) =>
+    model.yConfig?.color || model.config?.color;
+
+  publicAPI.getZAxisColor = (color) =>
+    model.zConfig?.color || model.config?.color;
+
+  const { setConfig } = publicAPI;
+
+  publicAPI.setConfig = (c) => {
+    if (setConfig(c)) {
       _debouncedUpdate();
       return true;
     }
     return false;
   };
```

Comment:
I think if you do `model._onConfigChanged = _debouncedUpdate();`, you don't have to overwrite `setConfig()`, don't you ?